content/en/api/users/send_invitation_email.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Send invitation email to a user
+type: apicontent
+order: 36.8
+external_redirect: /api/#send-invitation-email-to-a-user
+---
+
+## Send invitation email to a user
+
+Sends emails to one or more users inviting them to join the organization.
+
+**ARGUMENTS**:
```

Comment:
`data.relationships.user.data` can be an array of objects to invite multiple users in a single request.